File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromServer/LoggedInMessage.java
package org.demoth.aworlds.server2.api.messaging.fromServer;

import org.demoth.aworlds.server2.api.messaging.Message;

import java.util.Collection;
public class LoggedInMessage extends Message {
    public Collection<String> characters;

    public LoggedInMessage() {
    }

    public LoggedInMessage(Collection<String> characters) {
        this.characters = characters;
    }

}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromServer/LoggedInMessage.kt
package org.demoth.aworlds.server2.api.messaging.fromServer

import org.demoth.aworlds.server2.api.messaging.Message

class LoggedInMessage(var characters: Collection<String>) : Message() {

    override fun toString(): String {
        return "LoggedInMessage{" +
                "characters=" + characters +
                '}'.toString()
    }
}
